openpos-client-libs/projects/openpos-client-core-lib/src/lib/shared/status/status-details/status-details.component.html

```diff
@@ -1,14 +1,16 @@
 <div class="status-outer" responsive-class>
     <mat-card-title gdAreas="header backbutton" gdColumns="1fr auto" gdAlignColumns="center center">
         <div class="header" gdArea="header" gdColumns="auto 1fr" responsive-class>
-            <span>Status</span>
+            <span>Device Pairing</span>
```

Comment:
This should be internationalized.   @dankaste @kdelorey How would we do this?

Reply:
I added a `localize` pipe that does this for you given that there's a key on the server to match. Something along the lines of:

(Note I made this key up as an example)
```suggestion
            <span>{{status:details.dialog.title.deviceparing | localize}}</span>
```

Reply:
More details about the pipe can be found in #1081